Source/WebKit/WebProcess/WebPage/IPCTestingAPI.cpp

```diff
@@ -2475,15 +2475,19 @@ JSValueRef JSIPC::createStreamClientConnection(JSContextRef context, JSObjectRef
         return JSValueMakeUndefined(context);
     }
     auto connectionPair = IPC::StreamClientConnection::create(*bufferSizeLog2);
+    if (!connectionPair) {
+        *exception = createTypeError(context, "?????????"_s);
```

Comment:
you could prototype with following:

This near the createTypeError:
```
static JSValueRef createError(JSContextRef context, const String& message)
{
    JSC::JSLockHolder lock(toJS(context)->vm());
    return toRef(JSC::createError(toJS(context), message));
}
```

then use something like `createError(context, "Failed to create the connection"_s)`